3p/iframe-messaging-client.js

```diff
@@ -25,6 +25,8 @@ export class IframeMessagingClient {
   constructor(win) {
     /** @private {!Window} */
     this.win_ = win;
+    /** @private {?string} */
+    this.rtvVersion_ = (win.AMP_CONFIG && win.AMP_CONFIG.v) || null;
```

Comment:
The definition is [here](https://github.com/ampproject/amphtml/blob/master/src/mode.js#L41).